Code/Legacy/CryCommon/LegacyAllocator.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <CryCommon/LegacyAllocator.h>
+
+namespace AZ
+{
+    LegacyAllocator::pointer_type LegacyAllocator::Allocate(size_type byteSize, size_type alignment, int flags, const char* name, const char* fileName, int lineNum, unsigned int suppressStackRecord)
+    {
+        if (alignment == 0)
+        {
+            // Some STL containers, like std::vector, are assuming a specific minimum alignment. seems to have a requirement
```

Comment:
Not quite sure what this comment is saying with the "seems to have a requirement" at the end.  Maybe rephrase to "Some STL containers, like std::vector, seem to have a requirement where a specific minimum alignment will be chosen when the alignment is set to 0"?

Reply:
I actually just moved this code from the header to a cpp, will rephrase to your suggestion